PeerioTechnologies/peerio-desktop
# style: only Patient Rooms get bg colour, not Internal

#### Relevant info and issue/PR links 
mentioned in comments here:
https://www.figma.com/file/OmOPxwnmLmYKwF5VOEFXnCDQ/%237768-MedCryptor-Mobile%3A-Doctor's-Flow-in-app

#### Testing instructions  
This will only be visible on MedCryptor build. Only Patient Room should have the tinted orange chat background, not Internal Room. (On existing build, both have tint).

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
